lib/admin_form_builder.rb

```diff
@@ -0,0 +1,20 @@
+class AdminFormBuilder < GenericFormBuilder
```

Comment:
It might be clearer to call this a Bootstrap 3 form builder? As other than the input-md-4 there's no admin specific bits here. The form classes themselves are specific to Bootstrap 3 and may change in BS4.